python/lsst/daf/butler/cli/cliLog.py

```diff
@@ -48,8 +48,9 @@ class CliLog:
 
     defaultLsstLogLevel = lsstLog.FATAL if lsstLog is not None else None
 
-    longLogFmt = "%-5p %d{yyyy-MM-ddTHH:mm:ss.SSSZ} %c (%X{LABEL})(%F:%L)- %m%n"
-    normalLogFmt = "%c %p: %m%n"
+    lsstLog_longLogFmt = "%-5p %d{yyyy-MM-ddTHH:mm:ss.SSSZ} %c (%X{LABEL})(%F:%L)- %m%n"
+    lsstLog_normalLogFmt = "%c %p: %m%n"
+    pylog_longLogFmt = "%(asctime)s %(message)s"
```

Comment:
You don't think it's a good idea to emulate the `lsst.log` format and include things like `%(levelname)` and `%(name)` or even `%(filename)` and `%(lineno)`?
`asctime` is in local time by default.  We should either append timezone information (possibly by changing `logging.Formatter.default_time_format`) or use GMT/UTC consistently (possibly by changing `logging.Formatter.converter`).

Reply:
ok

Reply:
If you do use `gmtime()` to use GMT, you should make sure to include the "Z" timezone specifier in the format; it's more important that the timezone be clear than that any particular timezone be chosen.

Reply:
this seems to work if it seems ok to you(?)
```
pylog_longLogFmt = "%(asctime)s %(message)s"
...
            logging.basicConfig(level=logging.INFO,
                                format=cls.pylog_longLogFmt,
                                datefmt="%Y-%m-%dT%H:%M:%S%z")
```
example output:
```
2020-10-28T22:07:49-0700 Opening YAML config file: resource://lsst.daf.butler/configs/registry.yaml
2020-10-28T22:07:49-0700 Opening YAML config file: resource://lsst.daf.butler/configs/storageClasses.yaml
2020-10-28T22:07:49-0700 Opening YAML config file: resource://lsst.daf.butler/configs/datastore.yaml
```
The only difference between this and the lsst.log output is that the time does not include milliseconds. As far as I can tell the `time` format string does not offer a milliseconds or microseconds option.

lsst.log example output
```
2020-10-28T22:05:43-0700 Opening YAML config file: resource://lsst.daf.butler/configs/registry.yaml
2020-10-28T22:05:43-0700 Opening YAML config file: resource://lsst.daf.butler/configs/storageClasses.yaml
```

Reply:
That looks fine.  My understanding is that the logging Formatter is what adds on milliseconds; we can probably do without it.